app/models/passenger.rb

```diff
@@ -0,0 +1,20 @@
+class Passenger < ApplicationRecord
+
+  has_many :trips, -> { order(date: :desc) }, dependent: :nullify
+
+  validates :name, presence: { message: "You need to provide a name" }
+  validates :phone_num, presence: { message: "You need to provide a phone number" }
+  validates :phone_num, numericality: {message: "Please use only numbers in phone number"}
+
+  def total_cost
+
+    total = 0
+    trips.each do |trip|
+      total += trip.cost
+    end
+
+    return "$#{(total/100).round(2)}"
```

Comment:
I might argue against returning the formatting string here. The _value_ is what is important for this model method, and formatting that should probably be the responsibility of the view